content/es/developers/custom_checks/write_agent_check.md

```diff
@@ -1,44 +1,44 @@
 ---
+title: Escribir un check del Agent personalizado
 aliases:
-- /es/agent/faq/how-do-i-change-the-frequency-of-an-agent-check/
-- /es/agent/faq/agent-5-custom-agent-check/
-- /es/developers/write_agent_check/
+    - /agent/faq/how-do-i-change-the-frequency-of-an-agent-check/
```

Comment:
our automation adds the appropriate url prefix per translated site.  that's why it is missing here